docker/local/config.toml

```diff
@@ -10,12 +10,13 @@ pk = ""
 # Chain Public Address: 0x929adEcB790Eb4c8D371091475f0a824702ca1ef
 chainpk = ""
 
-naaddress = "0x86dbCA138C8aF6f9710d17F892bA788e87BeF000"
-vpaaddress = "0x9B955E99c325Be47e4De330F90d36D2E3D4778a9"
-caaddress = "0xC0FA32D0ca8067C9B5b4471Eb41299537E7B6a33"
+naaddress = "0xe32d4B5C5a80660710f6a2aD3cB1c11664138F34"
+vpaaddress = "0x4D1a804e1cE383D75032C52878553212992C98D7"
+caaddress = "0x0C9D79725afc344A388045235CD0B23eA4f0E838"
 
 # RPC provider docs: https://lotus.filecoin.io/lotus/developers/glif-nodes/#testnet-endpoint
 chainurl = "wss://wss.calibration.node.glif.io/apigw/lotus/rpc/v0"
+chainstartblock = 886696
 chainauthtoken = ""
```

Comment:
Out of scope, but do we need to specify these keys if they are just empty?

Reply:
No its not necessary. We could remove `chainauthtoken` from this config file and it should not affect the node at all